cfy_manager/utils/systemd.py

```diff
@@ -52,8 +52,9 @@ def configure(self, service_name, render=True):
         env_src = join(src_dir, sid)
         srv_src = join(src_dir, '{0}.service'.format(sid))
 
-        logger.debug('Deploying systemd EnvironmentFile...')
-        deploy(env_src, env_dst, render=render)
+        if exists(env_src):
```

Comment:
Why isn't that required?

Reply:
This was breaking when I removed the (empty) EnvironmentFile from this component (which wasn't referenced from the service file anyway)